frontend/components/dashboard/validator/subset/DashboardValidatorSubsetList.vue

```diff
@@ -105,12 +105,16 @@ function mapDutyLinks(
       path = '/slot/'
       break
   }
+
   if (path) {
     return dutyObjects.map(o => ({
       label: `${formatValue ? formatNumber(o) : o}`,
       to: `${path}${o}`,
     }))
   }
+  else if (props.category === 'pending' || props.category === 'exiting') {
+    return []
+  }
```

Comment:
would this also work?
```suggestion
  if (props.category === 'pending' || props.category === 'exiting') return []
 
  if (path) {
    return dutyObjects.map(o => ({
      label: `${formatValue ? formatNumber(o) : o}`,
      to: `${path}${o}`,
    }))
  }
```

Reply:
Good catch, I applied your suggestion